docs/_docs/Starters Academy (LDSSA)/04-Bootcamp-Structure.md

```diff
@@ -0,0 +1,83 @@
+regression and classification, and models used to tackle them. It is composed
+by 16 units, 14 mandatory, presented in the 2 days that compose the bootcamp, and 2
+optional. Together with the first 3 units you should have completed in the admission
+process, they compose the set of units of specialization #1. 
+
+| SLU   | Name                                     |  Presented at   | Mandatory |
+|-------|------------------------------------------|-----------------|-----------|
+| SLU01 | Pandas 101                               | Admissions      |  Yes      |
+| SLU02 | Subsetting Data in Pandas                | Admissions      |  Yes      |
+| SLU03 | Visualization with Pandas & Matplotlib   | Admissions      |  Yes      |
+| SLU04 | Basic Stats with Pandas                  | Bootcamp Day 1  |  Yes      |
+| SLU05 | Covariance & Correlation                 | Bootcamp Day 1  |  Yes      |
+| SLU06 | Dealing with Data Problems               | Bootcamp Day 1  |  Yes      |
+| SLU07 | Regression with Linear Regression        | Bootcamp Day 1  |  Yes      |
+| SLU08 | Metrics for Regression                   | Bootcamp Day 1  |  Yes      |
+| SLU09 | Classification with Logistic Regression  | Bootcamp Day 1  |  Yes      |
+| SLU10 | Metrics for Classification               | Bootcamp Day 1  |  Yes      |
+| SLU11 | Tree-Based Models                        | Bootcamp Day 2  |  Yes      |
+| SLU12 | Feature Engineering (aka Real Wold Data) | Bootcamp Day 2  |  Yes      |
+| SLU13 | Model Selection & Overfitting            | Bootcamp Day 2  |  Yes      |
+| SLU14 | Data Sufficiency & Selection             | Bootcamp Day 2  |  Yes      |
+| SLU15 | Ethics & Fairness                        | Bootcamp Day 2  |  Yes      |
+| SLU16 | Hyperparameter Tuning                    | Bootcamp Day 2  |  Yes      |
+| SLU17 | Workflow                                 | Bootcamp Day 2  |  Yes      |
+| SLU18 | Support Vector Machines (SVM)            | Not presented   |  No       |
+| SLU19 | k-Nearest Neighbors (kNN)                | Not presented   |  No       |
+
+During the days of the bootcamp, each presented unit will follow the following schedule:
+
+- 15 min presentation
+- 5 min questions
+- 40 min of working time on the unit
+
+However, it is not expected that in the 40 min of working time you complete all of the content, 
+it is just a first contact with the content. The bootcamp days are followed by 3 weeks where
+you are expected to review the materials and complete all units by yourself.
+
+
+
+### Time commitment
+
+Aside from the bootcamp weekend, you should be prepared to spend a recommended *10 hours per week* 
+on the bootcamp units throughout of the course of the 3 weeks given to complete them.
+
+
+### Bootcamp Evaluation Rules
+
+In order to pass the bootcamp:
+
+* The student must have been present in the bootcamp days
+* All mandatory units should have been submitted and score at least 16/20 until the deadline (see [below](#Calendar))
+
```

Comment:
below reference not working

Reply:
I didn't spot what's missing as I'm running late, but lmk if this isn't solved

Reply:
I changed these to just mention the section below, no need for a link